src/main/java/com/cloudbees/jenkins/plugins/bitbucket/client/branch/BitbucketCloudBranch.java

```diff
@@ -23,72 +23,64 @@
  */
 package com.cloudbees.jenkins.plugins.bitbucket.client.branch;
 
-import org.codehaus.jackson.annotate.JsonIgnoreProperties;
-import org.codehaus.jackson.annotate.JsonProperty;
-
 import com.cloudbees.jenkins.plugins.bitbucket.api.BitbucketBranch;
+import com.cloudbees.jenkins.plugins.bitbucket.client.repository.BitbucketCloudRepository;
+import org.codehaus.jackson.annotate.JsonCreator;
```

Comment:
imports not in alpha order, you touched the file, you own the violation ;-)

Reply:
done